launch/pacmod_game_control.launch

```diff
@@ -22,10 +22,11 @@
        POLARIS_RANGER
        LEXUS_RX_450H
        INTERNATIONAL_PROSTAR_122
+       FREIGHTLINER_CASCADIA
        VEHICLE_4
        VEHICLE_5
        VEHICLE_6 -->
-  <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" />
+  <arg name="pacmod_vehicle_type" default="POLARIS_GEM" />
```

Comment:
Please change this back to LEXUS_RX_450H. Changing the default launch parameters is a bad idea. Customers rely on them.